yoyodyne/evaluators.py

```diff
@@ -60,7 +60,7 @@ def accuracy(
         )[0].size()[0]
         # Gets the batch size (total_count).
         total_count = predictions.size(0)
-        return corr_count / total_count
+        return corr_count, total_count, corr_count / total_count
```

Comment:
You probably need to change the return type too.

I wonder if this should be a sort of `struct` or lightweight class rather than a tuple.

It looks like the third element of the tuple is never used.